README.md

```diff
@@ -1 +1,53 @@
-# H-ello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+> This is a project to display Hello Microverse.
+
+## Built With
+
+- HTML, CSS
+- Git
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Code Editor(VS Studio Code)
+Web Browser (Google Chrome recommended)
+
+How to Run
+To run the project do the following:
+
+- Clone the repository into a folder
+- Open the folder with your code editor (say; VS Code)
+- Install Live Server Extension on VS Code
+- Open the index.html file
+- Click on Live Server button on the base of your VS Code
+- Wait for a new tab to open
+- Voila! Your project is Live. You can procceed to make changes to the files
+
+## Author
+
+- GitHub: [@moses](https://github.com/Moses-chibuike)
+- LinkedIn: [moses](https://www.linkedin.com/in/ezechukwu-chibuike/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Your readme is missing certain key/important sections of the [Microverse Readme template](https://github.com/microverseinc/readme-template) like the `Tech Stack`, `Key Features`, `Future Features`, as well as most parts of the Getting Started section, etc. Note that it is a requirement to produce all sections of the readme template in this and all other subsequent projects you do in Microverse here. Kindly add them and make sure to customized to match your project credentials. :pray:

Reply:
Thanks @jerryowusu for the review, this was very clear